Extensions/PlatformBehavior/PlatformerObjectBehavior.cpp

```diff
@@ -44,38 +45,70 @@ PlatformerObjectBehavior::GetProperties(
     const gd::SerializerElement& behaviorContent) const {
   std::map<gd::String, gd::PropertyDescriptor> properties;
 
-  properties[_("Gravity")].SetGroup(_("Jump")).SetValue(
-      gd::String::From(behaviorContent.GetDoubleAttribute("gravity")));
-  properties[_("Jump speed")].SetGroup(_("Jump")).SetValue(
-      gd::String::From(behaviorContent.GetDoubleAttribute("jumpSpeed")));
+  properties[_("Gravity")]
+      .SetGroup(_("Jump"))
+      .SetType("Number")
+      .SetMeasurementUnit(gd::MeasurementUnit::GetPixelAcceleration())
```

Comment:
this isn't Newton because it's measured in pixels right?

Reply:
and also there is no mass handling in the platformer behavior